int64 secs
int64 nsecs
int32 seq
byte id
byte msgType
byte reTransmit
byte Ack